src/main/java/WebConnector.java

```diff
@@ -62,6 +62,36 @@ public static void run(Bank root) {
                 return new JSONObject(resp);
             }
         });
+        
+        // Administrator creates customer account
+        post("/accounts/create", (request, response) -> {
+        	try {        		
+        		Customer newCustomer = root.createCustomer(
+        			request.queryParams("firstName"),
+        			request.queryParams("lastName"),
+        			request.queryParams("email"),
+        			request.queryParams("password"),
+        			Double.parseDouble(request.queryParams("amount")), // initial balance
+        			Currency.valueOf(request.queryParams("currency"))
+        		);
+    
+        		ObjectMapper mapper = new ObjectMapper();
+
+                SimpleModule module = new SimpleModule();
+                module.addSerializer(Customer.class, new CustomerSerializer());
```

Comment:
This `CustomerSerializer` was added in order to create a DTO to return data back to the user (FE), e.g. it will contain a list of transactions. I'm not sure we need to return this information back to the administrator after the account creation. It's either we return back only status or serialized `Customer` class itself. For the second option, it could be tricky to implement cause we followed bidirectional integrity.